src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"reflect"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+	yaml "gopkg.in/yaml.v3"
+)
+
+func importConfig(
+	client *admin.AdminAPI,
+	filename string,
+	old_config admin.Config,
+	schema admin.ConfigSchema,
+	all bool,
+) (err error) {
+	readback_bytes, err := ioutil.ReadFile(filename)
+	out.MaybeDie(err, "Error reading file %s: %v", filename, err)
+	var readbackConfig admin.Config
+	err = yaml.Unmarshal(readback_bytes, &readbackConfig)
+	out.MaybeDie(err, "Error parsing edited config: %v", err)
+
+	type PropertyDelta struct {
```

Comment:
For function anonymous structs it matters a bit less, but generally I lowercase the first letter for structs that are not visible outside the package: `type properyDelta struct`